Mathlib/LinearAlgebra/Span/Defs.lean

```diff
@@ -228,6 +228,17 @@ theorem span_int_eq_addSubgroup_closure {M : Type*} [AddCommGroup M] (s : Set M)
 theorem span_int_eq {M : Type*} [AddCommGroup M] (s : AddSubgroup M) :
     (span ℤ (s : Set M)).toAddSubgroup = s := by rw [span_int_eq_addSubgroup_closure, s.closure_eq]
 
+theorem disjoint_of_disjoint_span (hst : Disjoint (span R s) (span R t)) :
```

Comment:
I'd be tempted to called this `_root_.Disjoint.of_span` and likewise for the `₀` version. Although I suppose it could conflict with `Idea.span`. :thinking: Do we really not already have this?